src/components/Details/Legislation/LegislationLatestVote.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, ReactNode } from "react";
+
+import Vote from "../../../models/Vote";
+import VoteBar from "./VoteBar";
+import H2 from "../../Shared/H2";
+import Details from "../../Shared/Details";
+import { getVoteDistribution } from "../../Shared/util/voteDistribution";
+import { strings } from "../../../assets/LocalizedStrings";
+import { VOTE_DECISION } from "../../../models/constants";
+
+const VOTE_BAR_HEIGHT = 16;
+
+export interface LegislationLatestVoteProps {
+  /** votes on the matter */
+  votes: Vote[];
+}
+
+const LegislationLatestVote: FC<LegislationLatestVoteProps> = ({
+  votes,
+}: LegislationLatestVoteProps) => {
+  const { inFavor, against, abstained } = getVoteDistribution(votes);
+  const votesForLabel: string = strings.number_approved.replace("{number}", `${inFavor.length}`);
+  const votesAgainstLabel: string = strings.number_rejected.replace(
+    "{number}",
+    `${against.length}`
+  );
+  const votesAbstainedLabel: string = strings.number_non_voting.replace(
+    "{number}",
+    `${abstained.length}`
+  );
+
+  const voteBars: ReactNode[] = [
+    <VoteBar
+      key={"inFavorVoteBar"}
+      status={VOTE_DECISION.APPROVE}
+      votes={inFavor}
+      percentage={(inFavor.length / votes.length) * 100}
+      height={VOTE_BAR_HEIGHT}
+      label={votesForLabel}
+    />,
+    <VoteBar
+      key={"againstVoteBar"}
+      status={VOTE_DECISION.REJECT}
+      votes={against}
+      percentage={(against.length / votes.length) * 100}
+      height={VOTE_BAR_HEIGHT}
+      label={votesAgainstLabel}
+    />,
+    <VoteBar
+      key={"abstainedVoteBar"}
+      status={VOTE_DECISION.ABSTAIN_NON_VOTING}
```

Comment:
Remind me, each muni can have different abstaining rules correct? Like some munis have abstains count as approve, some count abstain as reject, and then there are the "true abstain" like "i was in surgery and literally couldn't vote".

We are defaulting to using the third option here correct? In the future should this be customizable by muni?

Reply:
The status prop is really just used for color.  There will always be _only_ 3 bars: green/red/purple. Those three bars will merge all the relevant votes into themselves. E.g. There are 3 votes: VOTE_DECISION.ABSENT_APPROVE, VOTE_DECISION.ABSTAIN_APPROVE, VOTE_DECISION.APPROVE: they are ALL merged into the green bar. 

I prefer this over have a bunch of different bars which really mean the same thing (3 different green bars in the above example). If the user feels the need to see the _exact_ vote decision, we could have it in the expanded view).

Reply:
I agree that this is preferred than having 9 different bars. But this is a _perfect_ use case for the horizontal stacked bar. Being able to differentiate between different vote types _that added to bill approval_. I.e. one part of the bar is "true approve" and another part of the bar is "abstain (approve)" for example but they get slotted into the same bar.

Reply:
I added the exact vote decision as part of the expanded view so there can be no confusion.